osx/terminal-notifier.app/Contents/Info.plist

```diff
@@ -3,50 +3,61 @@
 <plist version="1.0">
 <dict>
 	<key>BuildMachineOSBuild</key>
-	<string>12E55</string>
+	<string>17A405</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleExecutable</key>
 	<string>terminal-notifier</string>
 	<key>CFBundleIconFile</key>
 	<string>Terminal</string>
 	<key>CFBundleIdentifier</key>
-	<string>nl.superalloy.oss.terminal-notifier</string>
+	<string>fr.julienxx.oss.terminal-notifier</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
 	<string>terminal-notifier</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.5.0</string>
+	<string>2.0.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleSupportedPlatforms</key>
+	<array>
+		<string>MacOSX</string>
+	</array>
 	<key>CFBundleVersion</key>
-	<string>8</string>
+	<string>15</string>
 	<key>DTCompiler</key>
 	<string>com.apple.compilers.llvm.clang.1_0</string>
 	<key>DTPlatformBuild</key>
-	<string>5A11365x</string>
+	<string>9A1004</string>
 	<key>DTPlatformVersion</key>
 	<string>GM</string>
 	<key>DTSDKBuild</key>
-	<string>13A538c</string>
+	<string>17A360</string>
 	<key>DTSDKName</key>
-	<string>macosx10.9</string>
+	<string>macosx10.13</string>
 	<key>DTXcode</key>
-	<string>0500</string>
+	<string>0901</string>
 	<key>DTXcodeBuild</key>
-	<string>5A11365x</string>
+	<string>9A1004</string>
 	<key>LSMinimumSystemVersion</key>
-	<string>10.8</string>
+	<string>10.10</string>
 	<key>LSUIElement</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2012 Eloy Durán. All rights reserved.</string>
+	<string>Copyright © 2012-2017 Eloy Durán, Julien Blanchard. All rights reserved.</string>
```

Comment:
isn't it a little much to add yourself to the authors based on this pr?

Reply:
I was looking at this PR, this the updated plist from terminal-notifier.
https://github.com/julienXX/terminal-notifier/blob/3ba9ce569e234062d09c8fd01c4be11e56a9fd1b/Terminal%20Notifier/Terminal%20Notifier-Info.plist#L30

The extra author is a contributor to terminal notifier, not the author of the PR :)